From: =?utf-8?q?Corentin_No=C3=ABl?= <corentin.noel@collabora.com>
Date: Tue, 5 Apr 2022 12:46:11 +0200
Subject: startup-notification: Specify the actual type in the "changed"
 signal

Allows to have type checks and the introspection to be aware of the actual type.

Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2362>

Origin: https://gitlab.gnome.org/GNOME/mutter/-/commit/c2b8582e
---
 src/core/startup-notification.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/startup-notification.c b/src/core/startup-notification.c
index d767d14..1a3dab5 100644
--- a/src/core/startup-notification.c
+++ b/src/core/startup-notification.c
@@ -682,7 +682,7 @@ meta_startup_notification_class_init (MetaStartupNotificationClass *klass)
                   META_TYPE_STARTUP_NOTIFICATION,
                   G_SIGNAL_RUN_LAST,
                   0, NULL, NULL, NULL,
-                  G_TYPE_NONE, 1, G_TYPE_POINTER);
+                  G_TYPE_NONE, 1, META_TYPE_STARTUP_SEQUENCE);
 
   g_object_class_install_properties (object_class, N_PROPS, sn_props);
 }
